packages/react-components/src/components/MessageThread.tsx

```diff
@@ -671,7 +679,9 @@ export const MessageThreadWrapper = (props: MessageThreadProps): JSX.Element =>
     /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
     attachmentOptions,
     /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
-    onRenderAttachmentDownloads
+    onRenderAttachmentDownloads,
+    /* @conditional-compile-remove(rich-text-editor) */
+    richTextEditor
```

Comment:
It should be `richTextEditor=false` to be consistent with a comment in the props interface (`@defaultValue `false`)